### DISCUSSION TITLE
Saving images in settings and loading them in Controller?

### DISCUSSION THREAD

**Discussion by JakeGilesPhillips** [area17/twill#2322] [General]
I've followed several guides and discussion posts about doing this and for some reason I just can't get it to work with my implementation. 

I have configured my `twill.php` file and am able to upload the image into my settings successfully, I just can't seem to retrieve it in the DisplayController.

I am using the method `TwillAppSettings::get('menu.menu.logo')` which works fine for other settings (types other than media).

Any ideas?

**Comment by ifox** [comment]
Hi @JakeGilesPhillips sorry for the delay. Did you figure this out? You need to retrieve the settings object and then call `->image` helpers like you would on any other Twill models.
